api/api.go

```diff
@@ -0,0 +1,53 @@
+package api
+
+import (
+	"io"
+	"net/http"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/jdxcode/netrc"
+)
+
+var (
+	// BaseURL is the root of the Section API
+	BaseURL = "https://aperture.section.io/api/v1"
```

Comment:
Would be handy to override via environment variable, especially for testing. One convention elsewhere has been `APERTURE_URI_PREFIX=https://aperture.section.io` but open to other ideas.

Reply:
Yeah that's a good call. 

Right now I am changing this variable inside the test suite, but I can see use cases where you might want to override it from the CLI.

Reply:
Fixed in e7fbf2e. 

Can be set by either: 

- Specifying the `--section-api-prefix` flag
- Specifying the `SECTION_API_PREFIX` environment variable